app/models/region.rb

```diff
@@ -80,11 +97,14 @@ def log_payload
 
   def initialize_path
     logger.info(class: self.class, msg: "got reparent_to: #{reparent_to.name}, going to initialize new path")
-    self.path = if reparent_to.path.present?
-      "#{reparent_to.path}.#{path_label}"
-    else
-      path_label
-    end
+
+    self.path =
+      if reparent_to.path.present?
```

Comment:
this formatting change makes me 😢 - is this your local editor preferences? I don't think standard does this

Reply:
Haha. I did this intentionally. I cannot bear the sight of an incorrectly aligned `if-else` ladder, it just hasn't ever been that way in the history of programming and somehow standard forces you to align the 3/4 of the ladder to the left and the `if` next to the RHS making for a completely arbitrary looking conditional.

I'd much rather see the `if-else` ladder, read like how it's supposed to:

```ruby
if <c>
**things
else
**other_things
end
```
or

```ruby
a = if <c>
    **things
    else
    **other_things
    end
```

Not,

```ruby
thing = if <c>
**thing
else
**other_things
end
```

I agree with [this](https://stackoverflow.com/a/13863289).

Reply:
I just use whatever standard auto formats to honestly. That is the point of standard - to not have to manually adjust formatting at all, ever, so we don't even have these discussions. 😁 😁